Userland/Libraries/LibWeb/WebAssembly/WebAssemblyTablePrototype.cpp

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/TypedArray.h>
+#include <LibWeb/WebAssembly/WebAssemblyTableObject.h>
+#include <LibWeb/WebAssembly/WebAssemblyTablePrototype.h>
+
+namespace Web::Bindings {
+
+void WebAssemblyTablePrototype::initialize(JS::GlobalObject& global_object)
+{
+    Object::initialize(global_object);
+    define_native_accessor("length", length_getter, {}, JS::Attribute::Enumerable | JS::Attribute::Configurable);
+    define_native_function("grow", grow, 1, JS::Attribute::Writable | JS::Attribute::Enumerable | JS::Attribute::Configurable);
+    define_native_function("get", get, 1, JS::Attribute::Writable | JS::Attribute::Enumerable | JS::Attribute::Configurable);
+    define_native_function("set", set, 1, JS::Attribute::Writable | JS::Attribute::Enumerable | JS::Attribute::Configurable);
```

Comment:
```suggestion
    define_native_function("set", set, 2, JS::Attribute::Writable | JS::Attribute::Enumerable | JS::Attribute::Configurable);
```

Reply:
`value` in `set()` is optional: https://webassembly.github.io/spec/js-api/#tables
`undefined set(unsigned long index, optional any value)`

Reply:
Yet apparently Firefox sets the length to 2
This seems very inconsistent to me :thinking:

Reply:
cool, let's keep it like this then. Firefox boog spotted:

![image](https://user-images.githubusercontent.com/19366641/132123816-5c81bfcb-d7d7-4384-bce8-98f7e7fef5e2.png)